src/cli/run.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+/* eslint-disable no-console */
+import type {IntrospectionQuery} from 'graphql/utilities/introspectionQuery';
+
+import {generateTypeFiles, processPragmas} from '../generateTypeFiles';
+import {processFiles} from '../parser/parse';
+import {resolveDocuments} from '../parser/resolve';
+import {schemaFromIntrospectionData} from '../schemaFromIntrospectionData';
+import {loadConfigFile} from './config';
+
+import {addTypenameToDocument} from 'apollo-utilities'; // eslint-disable-line flowtype-errors/uncovered
+
+import {execSync} from 'child_process';
+import fs, {readFileSync} from 'fs';
+import {buildClientSchema, type DocumentNode} from 'graphql';
+import {print} from 'graphql/language/printer';
+import {validate} from 'graphql/validation';
+import path from 'path';
+
+const findGraphqlTagReferences = (root: string): Array<string> => {
+    const response = execSync(
+        "git grep -I --word-regexp --name-only --fixed-strings 'graphql-tag' -- '*.js' '*.jsx'",
```

Comment:
This way of finding `gql` tags assumes we always import the tag from `graphql-tag`—i.e. that we don't re-export the tag from our own files. I think that's a reasonable assumption, but it might be worth documenting.